src/mca/rmaps/base/rmaps_base_map_job.c

```diff
@@ -598,14 +609,10 @@ void prte_rmaps_base_map_job(int fd, short args, void *cbdata)
                                                                 PRTE_BIND_TO_CORE);
                             }
                         } else {
-                            if (NULL
-                                != hwloc_get_obj_by_type(prte_hwloc_topology, HWLOC_OBJ_PACKAGE,
-                                                         0)) {
-                                prte_output_verbose(
-                                    5, prte_rmaps_base_framework.framework_output,
-                                    "mca:rmaps[%d] binding not given - using bypackage", __LINE__);
-                                PRTE_SET_DEFAULT_BINDING_POLICY(jdata->map->binding,
-                                                                PRTE_BIND_TO_PACKAGE);
+                            if (NULL != hwloc_get_obj_by_type(prte_hwloc_topology, HWLOC_OBJ_PACKAGE, 0)) {
```

Comment:
Shouldn't this be `bynuma` if `nprocs >2` to match that below (and in the `--help` doc)?

Reply:
Shouldn't this be `HWLOC_OBJ_NUMANODE` to match the default for `nprocs > 2`?